README.md

```diff
@@ -15,15 +15,40 @@ LPWC also supports irregular time intervals between time points collected in bio
 
 Installation
 ------------
-The first major release of LPWC will be added to CRAN.
-Until then, the easiest way to install LPWC is as follows:
+Prior to analyzing your data, the R package needs to be installed.
+
+The easiest way to install LPWC is through CRAN:
+
 ``` r
+install.packages("LPWC")
+```
+
+There are other additional ways to download LPWC.
+The first option is most useful if want to download a specific version of LPWC
+(which can be found at https://github.com/gitter-lab/LPWC/releases).
+``` r 
+devtools::install_github("gitter-lab/LPWC@vx.xx.x")
```

Comment:
Do we also need `library(devtools)` before this command, or is that not necessary?

Reply:
we dont need library(devtools) here